cocos2d/Platforms/Android/CCGLView.h

```diff
@@ -41,9 +41,8 @@ enum CCAndroidScreenMode {
     CCScreenScaledAspectFitEmulationMode
 };
 
-BRIDGE_CLASS("com.apportable.GLView")
-@interface CCGLView : GLView <CCDirectorView>
-
+BRIDGE_CLASS("org.cocos2d.CCGLView")
```

Comment:
is this correct? afaik there should be no `.java` files in cocos2d, and instead bridged from `GLActivityKit` - there is no CCGLView.java class to bridge with